File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/MedicinalProductManufactured.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.Resource;

public class MedicinalProductManufactured : DomainResource, IResource
{
    public required Base.Quantity Quantity { get; set; }
    public Base.ResourceReference[]? Ingredient { get; set; }
    public Base.ResourceReference[]? Manufacturer { get; set; }
    public Base.CodeableConcept? UnitOfPresentation { get; set; }
    public required Base.CodeableConcept ManufacturedDoseForm { get; set; }
    public Base.CodeableConcept[]? OtherCharacteristics { get; set; }
    public Base.ProdCharacteristic? PhysicalCharacteristics { get; set; }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/MedicinalProductManufactured.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import CodeableConcept
from base import ProdCharacteristic
from base import Quantity
from base import DomainResource
from base import Reference

@dataclass(kw_only=True)
class MedicinalProductManufactured(DomainResource):
    quantity: Quantity
    ingredient: Optional[List[Reference]] = field(default_factory=list)
    manufacturer: Optional[List[Reference]] = field(default_factory=list)
    unit_of_presentation: Optional[CodeableConcept] = None
    manufactured_dose_form: CodeableConcept
    other_characteristics: Optional[List[CodeableConcept]] = field(default_factory=list)
    physical_characteristics: Optional[ProdCharacteristic] = None